tests/integration/schema/simple_test.go

```diff
@@ -227,5 +227,26 @@ func TestSchemaSimpleErrorsGivenNonNullField(t *testing.T) {
 		},
 	}
 
-	testUtils.ExecuteTestCase(t, []string{"users"}, test)
+	testUtils.ExecuteTestCase(t, []string{"Users"}, test)
```

Comment:
thought: Would be nice to have an error raised upon incorrect / unknown Collection name.

Reply:
I think a better fix would to not require this param at all :)